src/index.js

```diff
@@ -63,18 +63,21 @@ $('.js_login-submit').on('click', function() {
       $('.customer_booked-room').removeClass('display_none')
       let name = customer.returnUserName()
       let money = customer.totalMoneySpent(bookings, rooms)
+      
       dom.userName(name)
       dom.welcome()
       dom.userTotalRevenueSpent(money)
     }
       populatePastBookings(userId, users, bookings)
       populateAllBooking(bookings, rooms, date)
-
 })
 
   function populatePastBookings(userId, users, bookings) {
-    const customer = new Customer(userId, users)
-    $('.main_customer-area').after(customer.returnBookedRooms(bookings));
+      const customer = new Customer(userId, users)
+      let x = customer.returnBookedRooms(bookings)
```

Comment:
Super nit-picky but I would rename the variable `x`. It'll make it easier if you have to come back and refactor/add functionality to this function.